smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptDependency.java

```diff
@@ -44,6 +44,7 @@
     AWS_SDK_HASH_NODE("dependencies", "@aws-sdk/hash-node", "3.6.1", true),
 
     AWS_SDK_URL_PARSER("dependencies", "@aws-sdk/url-parser", "3.6.1", true),
+    AWS_SDK_URL_PARSER_NATIVE("dependencies", "@aws-sdk/url-parser-native", "3.6.1", true),
```

Comment:
The package is committed mistakenly. The package has been removed in 3.13.0: https://github.com/aws/aws-sdk-js-v3/pull/2229